packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Send/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+
+import { actions } from 'data'
+
+import SelectAddress from './SelectAddress'
+
+// defines available steps to send amount
+export enum AvailableSteps {
+  SelectAddress = 0,
+  FirstStep = 1,
+  SecondStep = 2
+}
```

Comment:
You don`t need add to add =0, =1 etc
Just 
```
export enum AvailableSteps {
  SelectAddress,
  FirstStep,
  SecondStep
}
```

Reply:
yea, really good point